src/module/timedEffect/timedEffect.js

```diff
@@ -1,3 +1,31 @@
+import { DiceSFRPG } from "../dice.js";
+import RollContext from "../rolls/rollcontext.js";
+import { SFRPG } from "../config.js";
+
+function mkRemaining({ value, unit, enabled }) {
```

Comment:
Any particular reason this is a floating function rather than a method on `SFRPGTimedEffect`?

Reply:
Nothing in particular, I suppose. Personal preference. It didn't need to be a method, and it didn't fit as part of the TimedEffect's API, so I made it a file scoped utility function. I tend to err in that direction. Let me know if you'd prefer it to go anywhere else.

Reply:
Makes sense, so I'm happy to leave it as it is